docs/flush-tables-with-read-lock.md

```diff
@@ -22,18 +25,16 @@ Long-running queries with `FLUSH TABLES WITH READ LOCK` enabled can leave the se
 
 !!! note
    
-    All operations described in this section have no effect when [backup locks](https://docs.percona.com/percona-server/8.1/backup-locks.html) are used.
+    All operations described in this section have no effect when [Backup locks] are used.
 
-    Percona XtraBackup uses [Backup locks](https://docs.percona.com/percona-server/8.1/backup-locks.html)
-    where available as a lightweight alternative to `FLUSH TABLES WITH READ
+    Percona XtraBackup uses [Backup locks] where available as a lightweight alternative to `FLUSH TABLES WITH READ
     LOCK`. This operation automatically copies non-InnoDB data and avoids blocking DML queries that modify InnoDB tables.
 
-In order to prevent this from happening two things have been implemented:
+To prevent this from happening two things have been implemented:
 
- * xtrabackup waits for a good moment to issue the global lock
+* xtrabackup waits for a good moment to issue the global lock
```

Comment:
what is a "good moment"? Does this mean it waits for a minutes before issuing the global lock, or that it waits for an indefinite time before issuing the global lock?

Reply:
@altmannmarcelo, @satya-bodapati, please help define what a "good moment" is.

Reply:
updated